### ISSUE TITLE
Switch off action doesn't work on TYBAC-006 thermostat

### ISSUE THREAD

**Issue by Synochy** [nielsfaber/scheduler-component#403]
### Expected behavior
When I use the switch off action in a Scheduler card it will switch of my [TYBAC-006](https://www.zigbee2mqtt.io/devices/TYBAC-006.html#tuya-tybac-006) thermsotat.

{
    "child_lock": "UNLOCK",
    "current_heating_setpoint": 20,
    "deadzone_temperature": 1,
    "eco_mode": "OFF",
    "fan_mode": "low",
    "linkquality": 188,
    "local_temperature": 22,
    "local_temperature_calibration": -2,
    "manual_mode": "ON",
    "max_temperature": 25,
    "max_temperature_limit": 18,
    "min_temperature": 10,
    "min_temperature_limit": 30,
    "schedule_friday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_monday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_saturday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_sunday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_thursday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_tuesday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "schedule_wednesday": "06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0",
    "state": "ON",
    "system_mode": "heat",
    "valve": "CLOSE"
}

### Actual behavior
The scheduler send the service: climate.turn_off action but does nothing on this thermostat because it expected "state": "OFF"".

Is it possible to have the choice to use state instead of service to turn on and off a climate device ?
Is it possible to add the ECO mode too ?

**Comment by z-master42** [comment]
Can't you use the state entity to switch on and off?

**Comment by Synochy** [comment]
Yes I can use the state entity but then I can't use the scheduler crard anymore

**Comment by z-master42** [comment]
Why not? What type is the state entity? Switch? Boolean?

**Comment by Synochy** [comment]
It is a switch.
But I can't have only one schedule with my different temperature through the day and my switch off or the eco mode.
Or I did not find how to do it.

**Comment by Synochy** [comment]
Thanks to not close this issue

**Comment by nielsfaber** [comment]
If I understand the bug description correctly, your climate entity does not respond correctly to the climate.turn_off service.
Why would this be a bug on the side of scheduler? 
Can you share the value of the supported_features attribute of this climate entity?

> Is it possible to have the choice to use state instead of service to turn on and off a climate device ?

What is meant by this? Services (/actions) are used in HA to interact with entities and devices. There is no alternative.

**Comment by Synochy** [comment]
It is not really a bug, if you prefer you can call it a feature.
I would like scheduler to be able to send the service : 
- climate:state_off 
- climate:state_on
- climate:eco_mode_on
- climate:eco_mode_off

**Comment by z-master42** [comment]
I don't think this is possible because the thermostat in question doesn't support this at all and doesn't appear to be a "normal" thermostat.

Both the state entity and the eco mode entity are of binary type and are not part of the climate entity.
